neutronics/mcnp_inputs.py

```diff
@@ -0,0 +1,157 @@
+import math
+from string import Template
+import material_data as md
+
+def merge_comps(compA, compB):
+    """Merge two compositions for homogenization. Combine like isotopes when
+    necessary. This function takes two input compositions, and returns the union
+    of the two compositions.
+
+    Arguments:
+    ----------
+        compA (dict): first composition
+        compB (dict): second composition
+
+    Returns:
+    --------
+        compA (dict): merged comp. of A and B
+    """
+    for isotope in compB:
+        if isotope in compA:
+            compA[isotope] += compB[isotope]
+        else:
+            compA.update({isotope : compB[isotope]})
+
+    return compA
```

Comment:
Do you attempt merge composition or material ?

(In my mind composition are normalized where material are not)
if you are working with composition, maybe making sure they are normalized to 1 could avoid some problem in the future....

Reply:
If I followed correctly you don't want it to be normalized....

Reply:
I don't want it to be normalized. MCNP doesn't care if the materials are normalized, so I figured it didn't matter. Maybe I should change this for consistency going forward.